src/extensions/OrderBook.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity ^0.4.13;
+
+// NOTE: We're using address[100] to get back orderIDs from the orderFetcher as Solidity currently lacks
+// the ability to receive dynamic arrays from external contract calls. The list gets shrunk to the desired size
+// before returning
```

Comment:
The OrderBook function does return a dynamic array actually. The comment and note is about the OrdersFetcher interface defined and used here having a uint256[100] signature since we must call it from this contract to get order ids.